samples/SocketsSample/Startup.cs

```diff
@@ -23,6 +24,10 @@ public void ConfigureServices(IServiceCollection services)
             {
                 // Faster pings for testing
                 options.KeepAliveInterval = TimeSpan.FromSeconds(5);
+            })
+            .AddMessagePackProtocol(options =>
+            {
+                options.SerializationContext.DictionarySerlaizationOptions.KeyTransformer = DictionaryKeyTransformers.LowerCamel;
```

Comment:
Ugh, send a PR to fix this property 😄

Reply:
Depends on if we're going to stick with this library